src/__tests__/index.js

```diff
@@ -257,17 +257,20 @@ const shortTests = [
   [[{transform: 'translateX( 30px )'}], {transform: 'translateX( -30px )'}],
   [[{transform: 'translateX(30%)'}], {transform: 'translateX(-30%)'}],
   [
-    [{transform: 'translateY(30px) rotate(20deg) translateX(10px)'}],
-    {transform: 'translateY(30px) rotate(20deg) translateX(-10px)'},
+    [{transform: 'translateY(30px) scale(2) translateX(10px)'}],
+    {transform: 'translateY(30px) scale(2) translateX(-10px)'},
```

Comment:
I'm a little concerned about making changes to existing tests like this. Does this mean that the changes you made to the source will make these tests fail?

I'm sorry, but I'm not super familiar with the `transform` property to know whether this change would be acceptable for everyone else using the module. Based on your example, I think I'm good with this change, but I'm not sure what other unintended consequences this could have... I'd love to hear from another maintainer on this.

Reply:
Because rotation would now be flipped, these tests would have a different outcome. The idea here is to keep a transform value that does not flip in this center position. Alternatively, we could make sure the rotation flips here as well, but I think that is covered in a new test.

In this sense, it is a breaking change because something previously assumed to remain constant no longer behaves that way.

Reply:
I'm ok with minor breaking changes that are actually bug fixes. So I think that I'm ok with this change :+1: